lib/utils/zero-config.js

```diff
@@ -1,40 +1,38 @@
 function getEnvFromOptionsAndMode(mode, optionsObject) {
-    const optionsDevFlag = optionsObject.dev;
-    const optionsProdFlag = optionsObject.prod;
+    const { dev, prod } = optionsObject;
 
     if (process.env.NODE_ENV && process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'development') {
         return process.env.NODE_ENV;
     }
-    else if (optionsProdFlag) {
+    else if (prod) {
         return 'production';
     }
-    else if (optionsDevFlag) {
+    else if (dev) {
         return 'development';
-    } else if (!optionsDevFlag && !optionsProdFlag && mode) {
+    } else if (mode) {
         return mode;
     }
     return 'production';
 }
 
-module.exports = function setDefaultConfigBasedOnEnvironment(webpackObject) {
+function getConfigurations(options, outputOptions) {
```

Comment:
I think the old filename is more clear to what it is doing

Reply:
The old name is still there, this is just stubbed out function.

Reply:
Noticed, thx for ensuring!